README.md

```diff
@@ -143,6 +146,9 @@ Those that wish to contribute bug fixes, new features, refactorings and clean-up
 When you contribute, please take the following things into account:
 
 * Please cover all changes with unit tests and make sure that your code does not break any existing tests.
+    * To run the existing PHPUnit tests, [install PHPUnit](https://phpunit.de/getting-started.html) and then run this
```

Comment:
Actually, you should install the development-related composer packages using composer:

composer install

(omitting the --no-dev option then also installs the development packages).

PHPUnit will then be available as vendor/bin/phpunit .